asb-ballerina/Module.md

```diff
@@ -1,473 +1,262 @@
+                properties: {a: "propertyValue1", b: "propertyValue2"}
+            };
 
-### Step 9: Close Sender Connection
-You can now close the sender connection.
-```ballerina
-    checkpanic asbClient->closeSender(queueSender);
-```
+            asb:Message message = {
+                body: byteContent,
+                contentType: asb:TEXT,
+                timeToLive: timeToLive,
+                applicationProperties: applicationProperties
+            };
 
-### Step 10: Close Receiver Connection
-You can now close the receiver connection.
-```ballerina
-    checkpanic asbClient->closeReceiver(queueReceiver);
-```
+            check asbClient->send(queueSender, message);
+
+            check asbClient->closeSender(queueSender);
+        }
+    ```
+
+    Following is an example on how to receive messages from the Azure Service Bus using the client connector. You need to create a 
+    `queueReceiver` before receiving a message. Optionally you can provide the receive mode which is PEEKLOCK by default. You can find more information about the receive modes [here](https://docs.microsoft.com/en-us/java/api/com.microsoft.azure.servicebus.receivemode?view=azure-java-stable).
+
+    Receive a message from the Azure Service Bus
+
+    ```ballerina
+        public function main() returns error? {
+            handle queueReceiver = check asbClient->createQueueReceiver(queueName, asb:RECEIVEANDDELETE);
+
+            int serverWaitTime = 60; // In seconds
+
+            asb:Message|asb:Error? messageReceived = asbClient->receive(queueReceiver, serverWaitTime);
+
+            if (messageReceived is asb:Message) {
+                log:printInfo("Reading Received Message : " + messageReceived.toString());
+            } else if (messageReceived is ()) {
+                log:printError("No message in the queue.");
+            } else {
+                log:printError("Receiving message via Asb receiver connection failed.");
+            }
+
+            check asbClient->closeReceiver(queueReceiver);
+        }
+    ```
+
+    Following is an example on how to asynchronously listen to messages from the Azure Service Bus using the listener. 
+    You need to create a new listener instance before listening. Then, you need to create a service object with the service 
+    configuration specified using the `@asb:ServiceConfig` annotation and attach it to the listener. You need to give the 
+    connection string and the entity path of the queue we are to listen messages from. We can optionally provide the 
+    receive mode. Default mode is the PEEKLOCK mode. You can find more information about the receive modes 
+    [here](https://docs.microsoft.com/en-us/java/api/com.microsoft.azure.servicebus.receivemode?view=azure-java-stable). 
+    Finally, you can provide the service logic to execute when a message is received inside the onMessage remote function.
+
+    Listen to Messages from the Azure Service Bus
+
+    **NOTE:**
```

Comment:
Notes should follow the markdown syntax.

Reply:
Resolved and amended the changes. Thank you for the suggestion.